doc/tutorial/search/index.md

```diff
@@ -158,86 +158,138 @@ The following hits were found:
+configuration can only exists once within one search configuration. Trying to specify more than one hit configuration
+in one search configuration will fail at compile time with a static assertion.
+Sometimes the program you write requires to choose between different hit configurations depending on a user given
+program argument at runtime. To handle such cases you can also use the dynamic configuration seqan3::search_cfg::hit.
+This configuration object represents one of the four hit configurations mentioned previously and can be modified at
+runtime. The following snippet gives an example for this scenario:
+
+\snippet doc/tutorial/search/search_small_snippets.cpp hit_dynamic
+
+Note that the same rules apply to the dynamic as for the static hit configurations. Meaning, it can be added via the
+`|`-operator to the search configuration but cannot be combined with any other hit configuration.
 
-If you want to choose the hit strategy at runtime you can also use the dynamic configuration `seqan3::search_cfg::hit`.
+A closer look to the strata configuration reveals that it is initialised with an additional parameter called the
+`stratum`. The `stratum` can be modified even after it was added to the search configuration like the following example
+demonstrates:
 
-The `seqan3::search_cfg::strata` configuration element needs an additional parameter:
 \snippet doc/tutorial/search/search_small_snippets.cpp hit_strata
 
-If the best hit had an edit distance of 1, the strata strategy would report all hits with up to an edit distance of 3.
+We introduced here a new concept when working with the seqan3::configuration object, which is much like the access
+interface of a std::tuple. Concretely, it is possible to access the stored
+configuration using the `get<cfg_type>(cfg)` interface, where `cfg_type` it the name of the configuration type we like
+to access. The `get` interface returns a reference to the stored object that is identified by the given name. If you
+try to access an object which does not exist within the search configuration a static assert will be omitted at compile
+time, such that no invalid code can be generated.
+
+\note We need to use the expression `using seqan3::get;` before we can call the `get` interface in order to allow the
+      compiler to find the correct implementation of it based on the passed argument. This is related to how C++
+      resolves unqualified lookup of free functions in combination with function templates using an explicit template
+      argument such as the `get` interface does.
+
+So it remains the open question of what does the stratum actually do? In the above example, if the best hit found by
+the search for a particular query had an edit distance of 1, the strata strategy would report all hits with up to an
+edit distance of 3 before modifying the stratum and 2 after setting it to 1.
```

Comment:
```suggestion
edit distance of 3.
```
I'm not sure what you want to say?

Reply:
That is part of the example which you don't see in the review mode :). I make it two different sentences though.

Reply:
or just omit the confusion at all.